### ISSUE TITLE
Beta: Highlight current challenge in map?

### ISSUE THREAD

**Issue by moT01** [freeCodeCamp/freeCodeCamp#16902]
For some challenges (mainly the "introduction" challenges), I can't tell what challenge I'm currently on because the title isn't displayed in the description. Most of the challenges have the title at the top of the description so I can figure out what challenge it is. However, even when the title is displayed at the top I have trouble finding it on the map. 
<br>
![screenshot from 2018-03-15 18-38-17](https://user-images.githubusercontent.com/20648924/37496357-12f8f9cc-2880-11e8-86e9-504742bf32c8.png)
<br>
**I propose to highlight the currently displayed challenge in the map somehow.**

**Comment by QuincyLarson** [comment]
I agree. I think the easiest way to do this would be to put the current challenge in bold.

**Comment by love1024** [comment]
@QuincyLarson  the text is already bold so making it bold will have no effect. But we can make the background  highlight on selection. Any suggestions?
<img width="766" alt="selector" src="https://user-images.githubusercontent.com/19265795/38095988-6688e04c-338f-11e8-8c6c-cc0e301aa738.png">
